WcaOnRails/db/migrate/20170417072301_remove_description_from_teams.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class RemoveDescriptionFromTeams < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :teams, :description, :text
+    remove_column :teams, :name, :string
+    add_column :teams, :email, :string
+    add_column :teams, :rank, :integer
```

Comment:
I'm okay with ranks being tied.

I don't feel strongly about `display_rank` vs `rank`, but I'm going to leave this as `rank` for now because our events already have a rank field. If we want to rename all the `rank`s to `display_rank`s in a later PR, I'm okay with that.